server/src/main/java/com/paypal/selion/proxy/SeLionRemoteProxy.java

```diff
@@ -212,17 +216,6 @@ public void afterSession(TestSession session) {
                 appendMsgToCustomLog("Completed session #" + totalSessionsCompleted);
             }
             appendMsgToCustomLog("Total Number of slots used : " + getTotalUsed() + " on node :" + getId());
-            if (shutDownInProgress) {
-                appendMsgToCustomLog("Proceeding with shutdown of node " + getId());
-                if (getTotalUsed() <= 1) {
-                    shutdownNode();
-                    if (forceShutDown) {
```

Comment:
@sundaramrajendran @sebady Does force node shutdown (from the SeLion Grid Console) still work after this change?